src/Recorder/FilesystemRecorder.php

```diff
@@ -37,6 +38,11 @@ final class FilesystemRecorder implements RecorderInterface, PlayerInterface, Lo
      */
     private $filters;
 
+    /**
+     * @var LoggerInterface
+     */
+    protected $logger;
```

Comment:
this field is provided by LoggerAwareTrait. i'd rather not declare it - when we allow psr log 3.0 it will start to have a type declaration and this line would break.

Reply:
`LoggerAwareTrait` has a nullable `$logger` property, which PHPStan complains about. I can null-check the property so we can remove this redeclaration.

Reply:
ah, i see. i think the cleanest solution honestly would be to add a rule to ignore that specific phpstan complaint, as we know it is incorrect. the setLogger method of the trait has a typehint, so we also can't later null the property in any way. can you please add a phpstan.neon file with something like

```
parameters:
  ignoreErrors:
    - message: ...
      path: src/Recorder/FilesystemRecorder.php
```

Reply:
Done in #15.